js/ui/toast.js

```diff
@@ -146,13 +146,10 @@ const Toast = Overlay.inherit({
             },
             {
                 device: function(device) {
-                    const isPhone = device.deviceType === 'phone';
-                    const isAndroid = device.platform === 'android';
```

Comment:
Is there a reason to remove it? Please note - the new position config will be applied to all phones.

Reply:
I tested it in iphone and android. Should we specify only these platforms?

Reply:
fixed